### PR TITLE
doc/glossary.rst: define "Ceph Manager"

### PR DESCRIPTION
**Pull request by zdover23**
Add a definition for Ceph Manager (ceph-mgr).

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/glossary.rst [hunk 1]
```diff
@@ -59,6 +59,12 @@
                 with the Ceph System (for example: ``ceph.ko``, ``rbd.ko``).
 
 	Ceph Manager
+                The Ceph manager daemon (ceph-mgr) is a daemon that runs
```
Comment (anthonyeleven) [inline doc/glossary.rst:62]: s/is a daemon that//

File: doc/glossary.rst [hunk 2]
```diff
@@ -59,6 +59,12 @@
...
+                alongside monitor daemons to provide monitoring and interfacing
```
Comment (anthonyeleven) [inline doc/glossary.rst:63]: s/interfacing/interfaces/

**Review by anthonyeleven — APPROVED** [review]
Made two suggestions, approved so I'm not blocking

**Comment by zdover23** [comment]
> Made two suggestions, approved so I'm not blocking

I see your point in both cases, but I'm not going to make those changes just yet. Filling out the rest of the glossary is a tedious and tricky process, and I am trying to keep track of the exact locations from which I am drawing the text that I put into the glossary.

I expect to make a second pass over the whole glossary in December, and the purpose of that pass will be to put in more elegant phrases (like these). So: stick around for that.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/48764 - Quincy backport
https://github.com/ceph/ceph/pull/48765 - Pacific backport

**Pull request merged**